### PR TITLE
CLDR-10772 ST examples for month names, in appropriate format

### PR DESCRIPTION
**Pull request by pedberg-icu**
CLDR-10772

- [x] This PR completes the ticket.

Add ST examples for month names showing them in appropriate context: format months in a format with day number (currently winning format for skeleton yMMMMd), and stand-alone months in a format without (currently winning format for skeleton yMMMM)

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by pedberg-icu** [comment]
Once this is approved and tests pass, any TC member should feel free to merge so we can also review in smoketest.

File: tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java [hunk 1]
```diff
@@ -1094,6 +1096,71 @@ private String handleDayPeriod(XPathParts parts, String value) {
...
+        int skeletonIndex = (length.equals("wide")) ? 0 : 1;
+        if (!context.equals("format")) {
+            skeletonIndex += 2;
+        }
+        String checkPath =
+                "//ldml/dates/calendars/calendar[@type=\""
+                        + calendarId
+                        + "\"]/dateTimeFormats/availableFormats/dateFormatItem[@id=\""
+                        + skeletons[skeletonIndex]
+                        + "\"]";
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java:1123]: I wonder if we should have an XPathParts Builder designed for constructing XPaths in a loop with syntax checking (i.e. that the xpath remains valid).

**Review by srl295 — APPROVED** [review]

File: tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java (hunk 1 above)
Comment (pedberg-icu) [reply tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java:1123]: How many places would we use something like that?

**Comment by pedberg-icu** [comment]
Looks good in smoketest too.

**Pull request merged**